### DISCUSSION TITLE
Config kafka affinity  "strimzi.io/name" not working

### DISCUSSION THREAD

**Discussion by chanvoyage** [strimzi/strimzi-kafka-operator#5587] [Q&A] [answered]
i I doubt about use affinity  to improve reliability the strimzi.io/name ,  kafka cluster  does not   separate  worker node  the same as
zookeeper cluster

![image](https://user-images.githubusercontent.com/39899497/133896342-cf1d1a24-ddd2-408b-9353-fd04c28bb91f.png)
![image](https://user-images.githubusercontent.com/39899497/133896383-65db4763-92b1-4a85-a1cd-696af93e433c.png)

[kafka.txt](https://github.com/strimzi/strimzi-kafka-operator/files/7190314/kafka.txt)

![image](https://user-images.githubusercontent.com/39899497/133896680-b30142d3-f383-4e10-96a6-a17bce53792f.png)

**Comment by scholzj** [comment, marked as answer]
I don't think you use the right value of the label - I do not think the value you have in your configuration is the one which Strimzi is using on the pods, so it has nothing to apply the pod-affinity on.

**Reply by chanvoyage** [reply]
thank you scholzi 
            my value of pod-affinity  mismatch
